site/app/libraries/database/PostgresqlDatabaseQueries.php

```diff
@@ -373,7 +373,8 @@ public function getAverageForGradeable($g_id, $section_key, $is_team) {
 
         return ($this->course_db->getRowCount() > 0) ? new SimpleStat($this->core, $this->course_db->rows()[0]) : null;
     }
-    public function getGradeablesPastAndSection() {
+    public function getGradeablesPastAndSection($gradeable_id) {
```

Comment:
This feels like like an existing function with straight-forward functionality that a feature got tacked onto it because it was convenient.  It makes more sense to rename the function so it more clearly describes its own behavior, or to make another query for the additional information when needed (looking at the usage of the new features).

Reply:
Yes I certainly didn't help what I believe is an already confusing function.  Now I renamed it and described the new behavior, hopefully it is more clear.  As for keeping the old query and creating a new one, I don't believe this is necessary as this function was made for this specific purpose and isn't used anywhere else.  However, I can certainly change this if you believe there is a reason to leave the original.